mobify/platform-scaffold
# update loggedIn state in login and register commands

LoggedIn state doesn't get updated nav

## Changes
- update loggedIn state in login and register commands

## Todos

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- open menu and login then open menu to check account links
